Gem/Code/Source/AutoGen/NetworkCannonComponent.AutoComponent.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="NetworkCannonComponent"
+    Namespace="MultiplayerSample"
+    OverrideComponent="true"
+    OverrideController="true"
+    OverrideInclude="Source/Components/NetworkCannonComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <Include File="Source/MultiplayerSampleTypes.h"/>
+
+    <ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
+
+    <RemoteProcedure Name="OnHit"
+        InvokeFrom="Authority"
+        HandleOn="Client"
+        IsPublic="true"
+        IsReliable="true"
+        GenerateEventBindings="false"
+        Description="The cannon cannon made a hit; tell clients so they can play particle fx">
+    </RemoteProcedure>
+
+
+    <ArchetypeProperty Type="AZ::Vector3" Name="Velocity"  Init="AZ::Vector3(1.0f, 0.0f, 0.0f)" ExposeToEditor="true" Description="Cannon ball movement velocity."/>
```

Comment:
You will probably want to add a collision type and/or collision group to collide against only, so that you avoid self or another objects that you don't want to collide with.

Reply:
Collision group would be set on the shape collider, not the network component right?